README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/facebook/bistro.svg?branch=master)](https://travis-ci.org/facebook/bistro)
 
 This README is a very abbreviated introduction to Bistro. Visit
-http://facebook.github.io/bistro for a more structured introduction, and for the docs.
+http://facebook.github.io/bistro for a more structured introduction, and the docs.
```

Comment:
This is not a grammar fix, but a style preference. To me, having 2 "for" clauses is more clear. But if you feel passionate about this, do it your way.